ModPatches/Medieval Overhaul/Patches/Medieval Overhaul/ThingDefs_Misc/MO_Apparel_Armor.xml

```diff
@@ -1,46 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Patch>
 
-	<!--Padded Surcoat, Gambeson-->
-	<!--Surcoat:skin, Gambeson:middle-->
+	<!-- === Padded Surcoat, Gambeson === -->
 	<Operation Class="PatchOperationAdd">
-		<xpath>Defs/ThingDef[defName="DankPyon_Apparel_Padded_Surcoat" or defName="Dankpyon_Apparel_Gambeson"]/statBases</xpath>
+		<xpath>Defs/ThingDef[
+			defName="DankPyon_Apparel_Padded_Surcoat" or defName="Dankpyon_Apparel_Gambeson"
+			]/statBases</xpath>
 		<value>
-			<Bulk>15</Bulk>
-			<WornBulk>6</WornBulk>
+			<Bulk>10</Bulk>
+			<WornBulk>3</WornBulk>
+			<ArmorRating_Sharp>0.24</ArmorRating_Sharp>
+			<ArmorRating_Blunt>0.36</ArmorRating_Blunt>
 		</value>
 	</Operation>
 
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/ThingDef[defName="DankPyon_Apparel_Padded_Surcoat"]/statBases/Mass</xpath>
 		<value>
-			<Mass>2.3</Mass>
+			<Mass>2</Mass>
 		</value>
 	</Operation>
 
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/ThingDef[defName="Dankpyon_Apparel_Gambeson"]/statBases/Mass</xpath>
 		<value>
-			<Mass>3</Mass>
+			<Mass>2.3</Mass>
 		</value>
 	</Operation>
 
 	<!-- Added extra coverage -->
 	<Operation Class="PatchOperationAdd">
-		<xpath>Defs/ThingDef[defName="DankPyon_Apparel_Padded_Surcoat" or defName="Dankpyon_Apparel_Gambeson"]/apparel/bodyPartGroups</xpath>
+		<xpath>Defs/ThingDef[
+			defName="DankPyon_Apparel_Padded_Surcoat" or defName="Dankpyon_Apparel_Gambeson"
```

Comment:
To my understanding, surcoats are loose outer garmets used in large part for identification, and typically have no/very short sleeves.